consensus/polybft/block_builder.go

```diff
@@ -161,15 +161,15 @@ func (b *BlockBuilder) WriteTx(tx *types.Transaction) error {
 }
 
 // Fill fills the block with transactions from the txpool
-func (b *BlockBuilder) Fill() error {
+func (b *BlockBuilder) Fill() []*types.Receipt {
```

Comment:
Could we have instead a function in `BlockBuilder` to return the receipts at any time?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/852/commits/f9e4392a8fbcad2f61c0eb784b1d48f4a324e38c